_inc/client/components/settings-group/index.jsx

```diff
@@ -40,10 +39,15 @@ export const SettingsGroup = props => {
 				{
 					support && (
 						<div className="jp-module-settings__learn-more">
-							<Button borderless compact href={ support }>
-								<Gridicon icon="help-outline" />
-								<span className="screen-reader-text">{ __( 'Learn More' ) }</span>
-							</Button>
+							<InfoPopover screenReaderText={ __( 'Learn more' ) }>
+								<ExternalLink
+									icon={ true }
```

Comment:
We should get rid of the external link icon on these. Here's the issue for it: https://github.com/Automattic/jetpack/pull/6252